doc/rados/operations/operating.rst

```diff
@@ -352,7 +352,29 @@ To stop a Ceph daemon on another node, use the following syntax::
 	sudo service ceph -a stop osd.0
 
 
+Stopping an Entire Cluster
+==========================
 
+To stop an entire ceph cluster (for example, if you must physically move
+hosts in a data center) first set the noout flag to prevent OSDs from being
+removed::
+	
+	ceph osd set noout
+
+Then shutdown all the services in your cluster in the manner appropriate for your platform.
+
```

Comment:
I didn't include that specificity intentionally. The command you reference would work on an Ubuntu machine, but not other platforms. You can see the various platform-specific incantations at the following links:

http://ceph.com/docs/master/rados/operations/operating/#id1
http://ceph.com/docs/master/rados/operations/operating/#starting-all-daemons
http://ceph.com/docs/master/rados/operations/operating/#id7

My assumption is that if a user is reading this part of the documentation they would have already become familiar with the commands to start and stop the individual services in their deployment, or would be able to readily find them. Omitting all the possible variations keeps this section concise, and prevents the need for future updates if/when things change.